### ISSUE TITLE
Keyboard navigation is borked in Firefox and Safari

### ISSUE THREAD

**Issue by rhamilto** [patternfly/patternfly-next#2464]
1. Visit https://www.patternfly.org/v4/documentation/react/demos/pagelayout/expandable-nav  in Chrome.  
1. Use the tab key to navigate through the ui.
1. Visit https://www.patternfly.org/v4/documentation/react/demos/pagelayout/expandable-nav in Firefox or Safari.
1. Try to use the tab key to navigate through the ui.
1. Smack palm on forehead.

cc: @spadgett, @janwright73

**Comment by janwright73** [comment]
this issue is not funny, the authoring of the issue - very funny.  Thanks for creating it @rhamilto!

**Comment by seanforyou23** [comment]
Hi @rhamilto, I think it's actually a browser setting. For Firefox, you need to navigate to `about:config` and add a new integer preference (by right clicking, and selecting new integer), providing a preference for `accessibility.tabfocus` and setting its value to `7`.

For Safari, you can navigate to Preferences > Advanced > Universal Access, then check the box for pressing Tab to highlight each item on the webpage.

Agree, super weird this isn't the case by default. Let me know if you run into any issues enabling these, it should be all you have to do.
